Packs/AWS/Integrations/AWS/AWS.py

```diff
@@ -1809,6 +1800,14 @@ def main():  # pragma: no cover
         else:
             raise NotImplementedError(f"Command {command} is not implemented")
 
+    except ClientError as client_err:
+        # Catch ClientError at the main level and try to handle it
+        try:
+            account_id = args.get("account_id", "")
+            AWSErrorHandler.handle_client_error(client_err, account_id)
+        except DemistoException as handler_err:
```

Comment:
This is redundant, if there will be another exception in the handler, it will be caught by the except Exception